e2e/nvmeof.go

```diff
@@ -49,6 +49,41 @@ var _ = ginkgo.Describe("SPDKCSI-NVMEOF", func() {
 			})
 		})
 
+		ginkgo.It("Test multiple PVCs", func() {
+			multiTestPodName := "spdkcsi-test-multi"
+			ginkgo.By("create multiple pvcs and a pod with multiple pvcs attached, and check data persistence after the pod is removed and recreated", func() {
+				deployMultiPvcs()
+				deployTestPodWithMultiPvcs()
+				defer func() {
+					deleteMultiPvcsAndTestPodWithMultiPvcs()
+					if err := waitForTestPodGone(f.ClientSet, multiTestPodName); err != nil {
+						ginkgo.Fail(err.Error())
+					}
+					for _, pvcName := range []string{"spdkcsi-pvc1", "spdkcsi-pvc2", "spdkcsi-pvc3"} {
+						if err := waitForPvcGone(f.ClientSet, pvcName); err != nil {
+							ginkgo.Fail(err.Error())
+						}
+					}
+				}()
+				err := waitForTestPodReady(f.ClientSet, 3*time.Minute, multiTestPodName)
+				if err != nil {
+					ginkgo.Fail(err.Error())
+				}
+
+				err = checkDataPersistForMultiPvcs(f)
+				if err != nil {
+					ginkgo.Fail(err.Error())
+				}
+			})
+		})
+	})
+})
+
+var _ = ginkgo.Describe("CACHE-NODE-TEST", func() {
```

Comment:
Did you test this new changes?

Reply:
No, as we don't support caching nodes in the current release. So I just moved them out.

Reply:
I just moved out caching node tests to a separate tests, so that they don't interface with the current tests.

Reply:
Ah I see